ask/src/main/java/com/vistrav/ask/Ask.java

```diff
@@ -150,6 +151,8 @@ public void onReceive(Context lContext, Intent intent) {
             if (permissionObj != null) {
                 permissionObj.denied(deniedPermissions);
                 permissionObj.granted(grantedPermissions);
+                if(deniedPermissions.size()==0)
+                    permissionObj.onAllPermissionsGranted(true);
```

Comment:
this looks good but there is annotations as well if you can implement that one it will be good